src/core/environment.js

```diff
@@ -246,12 +246,18 @@ p5.prototype.cursor = function(type, x, y) {
  * frames per second (usual for movies) or above will be enough for smooth
  * animations. This is the same as setFrameRate(val).
  *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments returns
- * the current framerate. The draw function must run at least once before it will
- * return a value. This is the same as <a href="#/p5/getFrameRate">getFrameRate()</a>.
- *
- * Calling <a href="#/p5/frameRate">frameRate()</a> with arguments that are not
- * of the type Number or are non-positive also returns current framerate.
+ * Calling <a href="#/p5/frameRate">frameRate()</a> with no arguments or
+ * with arguments that are not of type Number or are non-positive returns
+ * an approximation of the current frame rate. The draw function must run at
+ * least once before it will return a value. This is the same as using
+ * <a href="#/p5/getFrameRate">getFrameRate()</a>.
+ *
+ * The returned frame rate value is technically the rate at which the draw 
```

Comment:
Btw just a heads up, I think the tests are failing due to some trailing spaces here and on a few other lines.